### PR TITLE
Theme for Note-app

### PR DESCRIPTION
**Pull request by MiheerTamkhane**
- Light / Dark theme
- Storing in local storage

### CODE REVIEW

File: src/frontend/contexts/ThemeContext.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { createContext, useContext, useState, useEffect } from "react";
+
+const ThemeContext = createContext();
+
+const ThemeProvider = ({ children }) => {
+  const [theme, setTheme] = useState(localStorage.getItem("THEME"));
+  console.log(theme);
```
Comment (manojasarsa) [inline src/frontend/contexts/ThemeContext.js]: You can remove console logs if no longer needed.

File: src/frontend/contexts/ThemeContext.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    <ThemeContext.Provider value={{ theme, setTheme }}>
+      {children}
+    </ThemeContext.Provider>
+  );
+};
+
+const useTheme = () => {
+  const context = useContext(ThemeContext);
+
+  if (context === undefined) {
```
Comment (manojasarsa) [inline src/frontend/contexts/ThemeContext.js:28]: Good way of handling error

**Review by manojasarsa — APPROVED** [review]
The code seems quite clean and readable.
Good to go!

**Review by gopalbbharadva — APPROVED** [review]
Nice UI and home page, Feature is also working fine, great work .

**Pull request merged**